scenarios/straight/agent_prefabs.py

```diff
@@ -47,15 +47,23 @@ def _single_act(self, obs):
 register(
     locator="pose-boid-agent-v0",
     entry_point=lambda **kwargs: AgentSpec(
-        interface=AgentInterface(action=ActionSpaceType.TargetPose, waypoints=True),
+        interface=AgentInterface(
+            action=ActionSpaceType.TargetPose,
+            waypoints=True,
+            done_criteria=DoneCriteria(collision=False),
+        ),
         agent_builder=PoseBoidAgent,
     ),
 )
 
 register(
     locator="trajectory-boid-agent-v0",
     entry_point=lambda **kwargs: AgentSpec(
-        interface=AgentInterface(action=ActionSpaceType.Trajectory, waypoints=True),
+        interface=AgentInterface(
+            action=ActionSpaceType.Trajectory,
+            waypoints=True,
+            done_criteria=DoneCriteria(collision=False),
```

Comment:
Why `done_criteria=DoneCriteria(collision=False)` ?

Reply:
I was testing out that example, the boid agents are more compelling if they do not revert to "SUMO" controlled vehicles when colliding.